queries.sql

```diff
@@ -13,3 +13,41 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Transaction: Update species column and rollback
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Transaction: Update species based on conditions
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] As pointed out by the previous reviewer, and stated in the project requirement, you need to verify the changes before you commit. :+1: :smile: :small_airplane:  You need to update your query to this effect. here's a guide:
```sh
-- Transaction: Update species based on conditions

BEGIN;
UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
UPDATE animals SET species = 'pokemon' WHERE species IS NULL;

-- Verify the changes
SELECT * FROM animals;

COMMIT;
SELECT * FROM animals;
```